CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Change Log
+
+## 3.0.0 (unreleased)
+
+**Features:**
+
+- added whitelisted to configuration (with defaults)
+- added blacklisted to configuration (with defaults)
```

Comment:
Please add a link to PR that regards this change and create one message instead of two.